cli/flox/src/commands/environment.rs

```diff
@@ -2055,7 +2067,12 @@ impl Update {
                     new_lockfile,
                     old_lockfile,
                     ..
-                } = self.update_manifest(flox, concrete_environment)?;
+                } = Dialog {
+                    message: "Updating environment...",
+                    help_message: None,
+                    typed: Spinner::new(|| self.update_manifest(flox, concrete_environment)),
+                }
```

Comment:
nonblocking: I think we should have one on `LockedManifest::update_global_manifest(&flox, self.inputs)` as well